### PR TITLE
[fix][doc] Enable zk when starting standalone in pulsar-sql doc

### PR DESCRIPTION
**Pull request by RobertIndie**
Master Issue: #16354

### Motivation

After https://github.com/apache/pulsar/pull/15478 gets merged, the zookeeper won't be enabled by default when starting the pulsar standalone. But the pulsar SQL depends on the zookeeper. We need to set`PULSAR_STANDALONE_USE_ZOOKEEPER=1` to enable zookeeper when using the pulsar SQL as mentioned in https://github.com/apache/pulsar/issues/16354#issuecomment-1173092956.

### Modifications

* Add `PULSAR_STANDALONE_USE_ZOOKEEPER=1` before running pulsar standalone in pulsar-sql doc

These changes only need to apply to the doc of pulsar 2.11 and later.

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `doc-not-needed` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### CODE REVIEW

**Review by MarvinCai — APPROVED** [review]

**Review by shibd — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]
Thanks!

File: site2/docs/sql-getting-started.md [hunk 1]
```diff
@@ -17,10 +17,17 @@ To query data in Pulsar with Pulsar SQL, complete the following steps.
...
 ```bash
 
-./bin/pulsar standalone
+PULSAR_STANDALONE_USE_ZOOKEEPER=1 ./bin/pulsar standalone
 
 ```
 
+:::note
+
+Starting the Pulsar standalone cluster from scratch doesn't enable zookeeper by default.
```
Comment (momo-jun) [inline site2/docs/sql-getting-started.md:26]:
```suggestion
Starting the Pulsar standalone cluster from scratch doesn't enable ZooKeeper by default. 
```

File: site2/docs/sql-getting-started.md [hunk 2]
```diff
@@ -17,10 +17,17 @@ To query data in Pulsar with Pulsar SQL, complete the following steps.
...
 
-./bin/pulsar standalone
+PULSAR_STANDALONE_USE_ZOOKEEPER=1 ./bin/pulsar standalone
 
 ```
 
+:::note
+
+Starting the Pulsar standalone cluster from scratch doesn't enable zookeeper by default. 
+However, the Pulsar SQL depends on the zookeeper. Therefore, we need to set `PULSAR_STANDALONE_USE_ZOOKEEPER=1` to enable the zookeeper.
```
Comment (momo-jun) [inline site2/docs/sql-getting-started.md:27]:
```suggestion
However, the Pulsar SQL depends on ZooKeeper. Therefore, you need to set `PULSAR_STANDALONE_USE_ZOOKEEPER=1` to enable ZooKeeper.
```

**Comment by Anonymitaet** [comment]
Thank you! Looking forward to more contributions! 😊 @RobertIndie

**Pull request merged**